Assets/WebGLTemplates/ETX/TemplateData/DetectWebGL.js

```diff
@@ -0,0 +1,26 @@
+
+//NB: 11/02/2017
```

Comment:
This is a good script, but one thing that is important to note is that this page will rest as an iframe within another page. As a result, a redirection request may be seen as malicious.

Would it be possible instead to setup a CAPI expose so that the platform can field this? Thanks